packages/wrangler/src/pages.tsx

```diff
@@ -1098,29 +1101,77 @@ export const pages: BuilderCallback<unknown, unknown> = (yargs) => {
       )
     )
     .command("project", false, (yargs) =>
-      yargs.command(
-        "list",
-        "List your Cloudflare Pages projects",
-        () => {},
-        async (args) => {
-          const config = readConfig(args.config as ConfigPath, args);
-          const accountId = await requireAuth(config);
-
-          const projects: Array<Project> = await listProjects({ accountId });
+      yargs
+        .command(
+          "list",
+          "List your Cloudflare Pages projects",
+          () => {},
+          async (args) => {
+            const config = readConfig(args.config as ConfigPath, args);
+            const accountId = await requireAuth(config);
+
+            const projects: Array<Project> = await listProjects({ accountId });
+
+            const data = projects.map((project) => {
+              return {
+                "Project Name": project.name,
+                "Project Domains": `${project.domains.join(", ")}`,
+                "Git Provider": project.source ? "Yes" : "No",
+                "Last Modified": project.latest_deployment
+                  ? timeagoFormat(project.latest_deployment.modified_on)
+                  : timeagoFormat(project.created_on),
+              };
+            });
+            render(<Table data={data}></Table>);
+          }
+        )
+        .command(
+          "create [name]",
+          "Create a new Cloudflare Pages project",
+          (yargs) =>
+            yargs
+              .positional("name", {
+                type: "string",
+                demandOption: true,
+                description: "The name of your Pages project",
+              })
+              .options({
+                "production-branch": {
+                  type: "string",
+                  // TODO: Should we default to the current git branch?
+                  default: "production",
+                  description:
+                    "The name of the production branch of your project",
+                },
+              }),
+          async (args) => {
+            const { "production-branch": productionBranch, name } = args;
+
+            if (!name) {
+              throw new FatalError("Must specify a project name.", 1);
+            }
 
-          const data = projects.map((project) => {
-            return {
-              "Project Name": project.name,
-              "Project Domains": `${project.domains.join(", ")}`,
-              "Git Provider": project.source ? "Yes" : "No",
-              "Last Modified": timeagoFormat(
-                project.latest_deployment.modified_on
-              ),
-            };
-          });
-          render(<Table data={data}></Table>);
-        }
-      )
+            const config = readConfig(args.config as ConfigPath, args);
```

Comment:
wait I missed this, pages reads from wrangler.toml now?

Reply:
This is just grabbing the account ID. I felt a bit iffy about it as well, so we can take that out if you'd prefer.

Some of our commands require an account ID and project name (e.g. `CF_ACCOUNT_ID=xyz wrangler pages publish --project=my-project`). It's not the end of the world, but it's a bit unwieldy to have to pass them in as args. Maybe this is time to add a Pages section to `wrangler.toml` or introduce a Pages config file?

Reply:
`pages.toml` (or some other format!) is probably the way to go here

Reply:
Let's ship this as in for now since the other commands also read this? 

I'll open another PR next week ish which moves all of these over together into the new config file.

Reply:
imo, I don't think you should land this and have people use this and then change it again. Why not keep using the env var till you do so?

Reply:
That's fair too. I meant that we could since we already landed other commands that get accountId from `wrangler.toml` already... 😅

Reply:
Oh that's right. Aight noted

Reply:
Yeah, while product decide what they want here, I'm just going to leave it as-is, and we can then replace it everywhere once we have a decision.